### ISSUE TITLE
Fastlane: short_description for uk-UA is too long

### ISSUE THREAD

**Issue by IzzySoft** [jamie-mh/AuthenticatorPro#719]
Just a hint: `short_description.txt` with Fastlane is limited to 80 chars, yours for `uk-UA` currently has 95 chars and thus will be truncated. So maybe you want to shorten it properly?

**Comment by IzzySoft** [comment]
Thanks a lot! If you want the corresponding snippet from my checker script to adapt it for your CI, just let me know.

**Comment by jamie-mh** [comment]
It doesn't change very often, I'll just keep an eye out for anything exceeding 80 chars from now on.

**Comment by IzzySoft** [comment]
Yes, that should be fine too. And should you miss one, I'll send another heads-up within 4 weeks or so :see_no_evil:

**Comment by IzzySoft** [comment]
This time it is Litvak, @jamie-mh:

    com.stratumauth.app/metadata/lv-LV/short_description.txt has 81 chars

Maybe change it to "Pirmkoda divpakāpju autorizācijas (2FA) lietotne ar šifrētiem dublējumiem"? The license already makes clear it is FOSS.

**Comment by IzzySoft** [comment]
@jamie-mh now it's

* build/com.stratumauth.app/metadata/lt-LT/short_description.txt has 110 chars

doesn't Crowdin have an option to hard-limit the length, as Weblate does?
